projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.8-a.9 - 2022-03-23
+### Enhancements
+- External Media: add Openverse as an external Media provider.
+- Jetpack: open the QR Post feature to public, and improve its logo sizing.
```

Comment:
I think this could be rephrased, announcing the feature, and in a `### Major Enhancements` changelog section.